app/_locales/en/messages.json

```
@@ -1647,6 +1656,16 @@
   "initialTransactionConfirmed": {
     "message": "Your initial transaction was confirmed by the network. Click OK to go back."
   },
+  "inputLogicEmptyState": {
+    "message": "Only enter a number that you're comfortable with the contract spending now or in the future. You can always increase the spending cap later."
```

@coreyjanssen is our content guideline to remove trailing periods from the ends of content?

```suggestion
    "message": "Only enter a number that you're comfortable with the contract spending now or in the future. You can always increase the spending cap later"
```